batch/batch/worker/worker.py

```diff
@@ -1987,6 +1987,12 @@ def __init__(self):
         self.password = None
 
 
+class CopyStepCredentials:
+    def __init__(self):
+        self.username = None
+        self.password = None
```

Comment:
Actually, one thing to change. There's no point to these `username` and `password` being here, can we remove these?

Reply:
We should relatedly add an assert in `_current_user_access_token` that `self.credentials` is not only present, but that it is an instance of `CloudUserCredentials`